docs/contents/releases.md

```diff
@@ -1,128 +1,159 @@
 # Releases
 
-We use the following different release statuses:
+> ***IMPORTANT***: ACK does *not* have a single release status or version.
+> Different components within the ACK project have different release cadences,
+> versions and statuses. Please read the information below before installing
+> any ACK component.
+
+## Semantic Versioning
+
+ACK is a collection of custom Kubernetes controllers, one for each supported
```

Comment:
ACK is a collection of custom Kubernetes controllers that allow you to instantiate and manage AWS APIs.